<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HelpBrowser</class>
 <widget class="QMainWindow" name="HelpBrowser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>703</width>
    <height>583</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RetroShare Help</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../images.qrc">
    <normaloff>:/images/logo/logo_24.png</normaloff>:/images/logo/logo_24.png</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout">
    <property name="spacing">
     <number>6</number>
    </property>
    <property name="leftMargin">
     <number>9</number>
    </property>
    <property name="topMargin">
     <number>9</number>
    </property>
    <property name="rightMargin">
     <number>9</number>
    </property>
    <property name="bottomMargin">
     <number>9</number>
    </property>
    <item>
     <widget class="QFrame" name="frmFind">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>450</width>
        <height>79</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>79</height>
       </size>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::NoContextMenu</enum>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout">
       <property name="leftMargin">
        <number>9</number>
       </property>
       <property name="topMargin">
        <number>9</number>
       </property>
       <property name="rightMargin">
        <number>9</number>
       </property>
       <property name="bottomMargin">
        <number>9</number>
       </property>
       <property name="spacing">
        <number>6</number>
       </property>
       <item row="0" column="0">
        <layout class="QHBoxLayout">
         <property name="spacing">
          <number>6</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QToolButton" name="btnCloseFind">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <property name="contextMenuPolicy">
            <enum>Qt::NoContextMenu</enum>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../../images.qrc">
             <normaloff>:/images/button_cancel.png</normaloff>:/images/button_cancel.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>22</width>
             <height>22</height>
            </size>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonIconOnly</enum>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label">
           <property name="contextMenuPolicy">
            <enum>Qt::NoContextMenu</enum>
           </property>
           <property name="text">
            <string>Find:</string>
           </property>
           <property name="buddy">
            <cstring>lineFind</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineFind">
           <property name="contextMenuPolicy">
            <enum>Qt::NoContextMenu</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="btnFindPrev">
         <property name="contextMenuPolicy">
          <enum>Qt::NoContextMenu</enum>
         </property>
         <property name="text">
          <string>Find Previous</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>:/images/22x22/go-up-search.png</normaloff>:/images/22x22/go-up-search.png</iconset>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="btnFindNext">
         <property name="contextMenuPolicy">
          <enum>Qt::NoContextMenu</enum>
         </property>
         <property name="text">
          <string>Find Next</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>:/images/22x22/go-down-search.png</normaloff>:/images/22x22/go-down-search.png</iconset>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="3">
        <layout class="QHBoxLayout">
         <property name="spacing">
          <number>6</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <spacer>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QCheckBox" name="chkbxMatchCase">
           <property name="contextMenuPolicy">
            <enum>Qt::NoContextMenu</enum>
           </property>
           <property name="text">
            <string>Case sensitive</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="chkbxWholePhrase">
           <property name="contextMenuPolicy">
            <enum>Qt::NoContextMenu</enum>
           </property>
           <property name="text">
            <string>Whole words only</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="contextMenuPolicy">
       <enum>Qt::NoContextMenu</enum>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QTabWidget" name="tabsMenu">
       <property name="contextMenuPolicy">
        <enum>Qt::NoContextMenu</enum>
       </property>
       <property name="tabPosition">
        <enum>QTabWidget::North</enum>
       </property>
       <property name="tabShape">
        <enum>QTabWidget::Rounded</enum>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tabTOC">
        <property name="windowTitle">
         <string>RetroShare Help</string>
        </property>
        <property name="windowIcon">
         <iconset>
          <normaloff>:/images/help.png</normaloff>:/images/help.png</iconset>
        </property>
        <attribute name="title">
         <string>Contents</string>
        </attribute>
        <layout class="QVBoxLayout">
         <property name="spacing">
          <number>6</number>
         </property>
         <property name="leftMargin">
          <number>6</number>
         </property>
         <property name="topMargin">
          <number>6</number>
         </property>
         <property name="rightMargin">
          <number>6</number>
         </property>
         <property name="bottomMargin">
          <number>6</number>
         </property>
         <item>
          <widget class="QTreeWidget" name="treeContents">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
             <horstretch>1</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="focusPolicy">
            <enum>Qt::WheelFocus</enum>
           </property>
           <property name="contextMenuPolicy">
            <enum>Qt::NoContextMenu</enum>
           </property>
           <property name="selectionBehavior">
            <enum>QAbstractItemView::SelectItems</enum>
           </property>
           <property name="uniformRowHeights">
            <bool>true</bool>
           </property>
           <column>
            <property name="text">
             <string>Help Topics</string>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tabSearch">
        <attribute name="title">
         <string>Search</string>
        </attribute>
        <layout class="QGridLayout">
         <property name="leftMargin">
          <number>6</number>
         </property>
         <property name="topMargin">
          <number>6</number>
         </property>
         <property name="rightMargin">
          <number>6</number>
         </property>
         <property name="bottomMargin">
          <number>6</number>
         </property>
         <property name="spacing">
          <number>6</number>
         </property>
         <item row="2" column="1">
          <layout class="QHBoxLayout">
           <property name="spacing">
            <number>6</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <spacer>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="btnSearch">
             <property name="contextMenuPolicy">
              <enum>Qt::NoContextMenu</enum>
             </property>
             <property name="text">
              <string>Search</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_2">
           <property name="contextMenuPolicy">
            <enum>Qt::NoContextMenu</enum>
           </property>
           <property name="text">
            <string>Searching for:</string>
           </property>
           <property name="buddy">
            <cstring>lineSearch</cstring>
           </property>
          </widget>
         </item>
         <item row="3" column="0" colspan="2">
          <widget class="QTreeWidget" name="treeSearch">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
             <horstretch>1</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="focusPolicy">
            <enum>Qt::WheelFocus</enum>
           </property>
           <property name="contextMenuPolicy">
            <enum>Qt::NoContextMenu</enum>
           </property>
           <property name="selectionBehavior">
            <enum>QAbstractItemView::SelectItems</enum>
           </property>
           <property name="uniformRowHeights">
            <bool>true</bool>
           </property>
           <column>
            <property name="text">
             <string>Found Documents</string>
            </property>
           </column>
          </widget>
         </item>
         <item row="1" column="0" colspan="2">
          <widget class="QLineEdit" name="lineSearch">
           <property name="contextMenuPolicy">
            <enum>Qt::NoContextMenu</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="HelpTextBrowser" name="txtBrowser">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>100</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="contextMenuPolicy">
        <enum>Qt::NoContextMenu</enum>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="contextMenuPolicy">
    <enum>Qt::NoContextMenu</enum>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionBack"/>
   <addaction name="actionForward"/>
   <addaction name="actionHome"/>
   <addaction name="actionFind"/>
   <addaction name="separator"/>
   <addaction name="actionClose"/>
  </widget>
  <action name="actionBack">
   <property name="icon">
    <iconset resource="../../icons.qrc">
     <normaloff>:/icons/png/arrow-left.png</normaloff>:/icons/png/arrow-left.png</iconset>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
   <property name="statusTip">
    <string>Move to previous page (Backspace)</string>
   </property>
   <property name="shortcut">
    <string>Backspace</string>
   </property>
  </action>
  <action name="actionForward">
   <property name="icon">
    <iconset resource="../../icons.qrc">
     <normaloff>:/icons/png/arrow-right.png</normaloff>:/icons/png/arrow-right.png</iconset>
   </property>
   <property name="text">
    <string>Forward</string>
   </property>
   <property name="statusTip">
    <string>Move to next page (Shift+Backspace)</string>
   </property>
   <property name="shortcut">
    <string>Shift+Backspace</string>
   </property>
  </action>
  <action name="actionHome">
   <property name="icon">
    <iconset resource="../../icons.qrc">
     <normaloff>:/icons/png/home.png</normaloff>:/icons/png/home.png</iconset>
   </property>
   <property name="text">
    <string>Home</string>
   </property>
   <property name="statusTip">
    <string>Move to the Home page (Ctrl+H)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
  <action name="actionFind">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../images.qrc">
     <normaloff>:/images/find.png</normaloff>:/images/find.png</iconset>
   </property>
   <property name="text">
    <string>Find</string>
   </property>
   <property name="iconText">
    <string>Find</string>
   </property>
   <property name="toolTip">
    <string>Find</string>
   </property>
   <property name="statusTip">
    <string>Search for a word or phrase on current page (Ctrl+F)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="icon">
    <iconset resource="../../icons.qrc">
     <normaloff>:/icons/png/exit2.png</normaloff>:/icons/png/exit2.png</iconset>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
   <property name="statusTip">
    <string>Close Vidalia Help</string>
   </property>
   <property name="shortcut">
    <string>Esc</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>HelpTextBrowser</class>
   <extends>QTextBrowser</extends>
   <header>gui/help/browser/helptextbrowser.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>tabsMenu</tabstop>
  <tabstop>btnCloseFind</tabstop>
  <tabstop>lineFind</tabstop>
  <tabstop>btnFindNext</tabstop>
  <tabstop>btnFindPrev</tabstop>
  <tabstop>chkbxMatchCase</tabstop>
  <tabstop>chkbxWholePhrase</tabstop>
  <tabstop>treeContents</tabstop>
  <tabstop>txtBrowser</tabstop>
  <tabstop>lineSearch</tabstop>
  <tabstop>btnSearch</tabstop>
  <tabstop>treeSearch</tabstop>
 </tabstops>
 <resources>
  <include location="../../images.qrc"/>
  <include location="../../icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionFind</sender>
   <signal>triggered()</signal>
   <receiver>lineFind</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>200</x>
     <y>108</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnCloseFind</sender>
   <signal>clicked()</signal>
   <receiver>actionFind</receiver>
   <slot>toggle()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>34</x>
     <y>107</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionFind</sender>
   <signal>toggled(bool)</signal>
   <receiver>frmFind</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>243</x>
     <y>108</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionClose</sender>
   <signal>triggered()</signal>
   <receiver>HelpBrowser</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>317</x>
     <y>248</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineFind</sender>
   <signal>returnPressed()</signal>
   <receiver>btnFindNext</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>293</x>
     <y>108</y>
    </hint>
    <hint type="destinationlabel">
     <x>539</x>
     <y>108</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineSearch</sender>
   <signal>returnPressed()</signal>
   <receiver>btnSearch</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>153</x>
     <y>245</y>
    </hint>
    <hint type="destinationlabel">
     <x>244</x>
     <y>277</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
